README.CN.md

```diff
@@ -0,0 +1,524 @@
+| 2 - 贡献者  | 1 – 1.5      |
+| 3 - 参与者  | .5 – 1       |
+| 4 - 观望者     | < .5         |
+
+根据这些范围，得出的级别分布如下所示：
+
+[![Line chart showing Orbit Levels over time](https://res.cloudinary.com/dzello/image/upload/v1600473812/orbit/Orbit_Level_Chart.png)](https://res.cloudinary.com/dzello/image/upload/v1600473812/orbit/Orbit_Level_Chart.png)
+
+## 按照层级划分的活动
+
+通过按轨道层级组织活动清单，可以轻松地找到向每个成员提供的机会。 这是一个例子：
+
+| Activity                | 对应到轨道层级 |
+| ----------------------- | :------------------: |
+| 参与会议   |         所有层级          |
+| 答复论坛问题    |         所有层级          |
+| 发表主题演讲         |       轨道 1        |
+| 组织meetup        |       轨道 1        |
+| 在meetup上发表演讲       |       轨道 2        |
+| 提交一个PR     |       轨道 2        |
+| 在论坛发表议题       |       轨道 2        |
+| 在论坛分享项目 |       轨道 3        |
+| 参加社区活动            |       轨道 3        |
+| 获得产品积分 |       轨道 3        |
+| 阅读博客文章         |       轨道 4        |
+| 订阅新闻 |       轨道 4        |
+| 在微博关注我们    |       轨道 4        |
+
+
+这些活动并非仅限于轨道级别：布道者希望订阅新闻通讯，并在微博上关注我们。 但是，**当我们希望找到一个可以提供给布道者的活动时，我们可以寻找标有“轨道1”的活动，例如发表主题演讲或组织聚会**。
+
+## 如何晋升到下一个层级
+
+当你确定了一个准备好进入下一个轨道级别的社区成员时，这是一个充满欢乐的时刻。 在轨道模型中，我们称这个时刻为晋升。
+
+💡 促销可以是您与成员庆祝的实际时刻，也可以是您在社区数据库中记录下来的时刻。
+
+度量成员的热情将帮助你确定何时该开始向他们提供合适下一个轨道层级参与的活动和应承担的责任。 如果他们完成了这些活动并且似乎准备做更多的事情，您就会知道是时候晋升他们到下一层级了。
+
+# 轨道模型关键指标
+
+下面是使用轨道模型时要衡量的最重要指标的摘要。
+
+| 指标         | 应用于                            | 计算方法                              | 指标的用途                                     |
+| -------------- | ------------------------------------- | ---------------------------------------- | ------------------------------------------------ |
+| 向心力        | 整个社区, 轨道层级, 群组 | Sum(热情 \* 影响力)        | 整体描述社区成员增量和参与度的晴雨表      |
+| 层级大小     | 轨道层级 4, 3, 2, 1           | 每个层级成员的数量                | 社区成员的均衡性                    |
+| 轨道层级    | 社区成员                                | 由社区核心团队评定，参考成员的热情值        | 描述社区成员应在社区处于的位置           |
+| 热情           | 社区成员                                | 成员最近参与的活动评分     | 描述社区可以让成员做出何种贡献 |
+| 影响力          | 成员                                | 成员影响力分数的总和     | 描述每个社区成员吸引其他人的能力           |
+| 活动分数 | 社区成员                                | 完成活动后分数的汇总 | 描述社区成员的参与度            |
+
+# 完整的案例
+
+[Orbit Model Airtable模板](https://airtable.com/shrOH6UzOdzZICXnJ) 包含示例成员数据，公式和图形，它们向你展示轨道模型的各个部分是如何组合在一起的。
+
+阅读[博客全文](https://orbit.love/blog/introducing-the-orbit-model-airtable-template)，以了解模板的各个部分如何工作并制作自己的副本。
+
+# 其他主题
+
+## 群组
+
+大型社区通常由一组较小的社区组成。 在轨道模型中，这些子社区称为群组。 群组可以围绕人，地方和事物形成。 例如：
+
+- 人 - 有影响力的社区成员组成的群组； 比如：大使
+- 地方 - 地理位置群组，例如城市，州或国家/地区
+- 事物 - 兴趣群组，依据独特的信仰或观点
+
+与更大社区相比，群组成员可以更轻松地与本组中的人联系。 与特定人员，地点或事物相关联的成员在成为协作者时也更容易。
+
+💡 在轨道模型中，可以将群组可视化为从轨道1延伸到轨道4的三角形扇形切片。 每个小组都有自己的布道者，贡献者，参与者和观望者。
+
+**增强社区向心力的技巧：确定活跃或潜在的群组，帮助他们扩大影响力，吸引新成员加入该群体.**
+
+## 轨道模型和漏斗模型的对比
+
+轨道模型和营销领域的漏斗模型之间有什么关系？ 这是一个大家常问的问题，也是一个很好的问题。 出售产品并拥有社区的公司通常应同时采用这两种模型。 但是他们主要的区别是什么？
+
+漏斗模型是为公司“获取价值”的模型。 关键衡量指标是不同阶段的转化率，最终结果是转化收益。
+
+**轨道模型是为社区创造价值。**我们衡量社区中正在创造多少价值以及由谁来创造价值，以便为社区成员提供正确的激励和机会。
+
```

Comment:
Missing two paragraphs
“
As an example (using the Orbit community), some of our community members work in gaming, and so we might consider options for connecting them to one another or creating a space for them to talk about Orbit and gaming.

Taking a developer tool as another example, it would be possible for community facilitators to group community members by SDK, and then further invite the most active members to lead each particular SDK group.
”

Reply:
谢谢 确实漏掉了两段